src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/BindToInputBindingProvider.cs

```diff
@@ -115,51 +113,54 @@ public Type GetDefaultType(Attribute attribute, FileAccess access, Type requeste
             return null;
         }
 
-        private class ExactBinding<TUserType> : BindingBase<TAttribute>
+        private class ExactBinding : BindingBase<TAttribute>
         {
-            private readonly Func<object, object> _buildFromAttribute;
```

Comment:
>Func<object, object>  [](start = 29, length = 21)

Good to be getting rid of these adhoc signatures. Everything that does a converter is a FuncAsyncConverter.